client/src/bastionlab/polars/remote_polars.py

```diff
@@ -313,64 +315,126 @@ def join_asof(
             ),
         )
 
-    def histplot(self: LDF, col_x: str, col_y: str, bins: int = 10, **kwargs):
+    def histplot(
```

Comment:
Tutorials and all is good, but personally, I would really like to see a docstring associated to the histplot method. You can use as an example what we did on BlindAI: https://github.com/mithril-security/blindai/blob/075929dd5e030983a0c153e1c6052297eca6f53a/client/blindai/client.py#L602

Reply:
-> Done docstring for all visualize/Facet methods ✅ 
-> Also implemented kwargs for Facet class properly as realised I was not forwarding kwargs to subplots() in Facet functions